python/ray/serve/tests/test_replica_scheduler.py

```diff
@@ -154,12 +207,12 @@ async def test_two_replicas_available_then_one(pow_2_scheduler, fake_query):
 
     s.update_replicas([r1, r2])
 
-    for _ in range(100):
+    for _ in range(10):
```

Comment:
curious why do we reduce the number?

Reply:
the number was arbitrary from the start and the test started taking >1s to run :)